less/home-page.less

```diff
@@ -72,4 +70,93 @@
       }
     }
   }
-}// home
\ No newline at end of file
+}
+
+button{
+  .buttonTag
+
+}
+&:hover{
+    background-color: white;
+    color: black;
```

Comment:
You could actually have included this animation in the `buttonTag` mixin, this way you can preserve the same button styling and even hover animations in that mixin and have it handle almost (if not) all the button styling.